bundles/core/org.eclipse.smarthome.core.thing/src/main/java/org/eclipse/smarthome/core/thing/ThingUID.java

```diff
@@ -168,6 +169,20 @@ public ThingTypeUID getThingTypeUID() {
     }
 
     /**
+     * Returns the thing id.
+     *
+     * @return thing id
+     */
+    public String getThingId() {
```

Comment:
This method does not make much sense to me as "thingId" does not seem to be what it returns - after all, there is already a getId() method available, that returns the id of the thing.
I will have a look and see if I can come up with some alternative...